local/kaltura/js/kea_resize.js

```diff
@@ -0,0 +1,15 @@
+(function() {
+    window.addEventListener('message', function (e) {
+        try {
+            var postMessageData = e.data;
+            if (postMessageData.subject === "lti.frameResize") {
+                var height = postMessageData.height;
+                var lti = document.querySelector("#contentframe");
+                lti.style.height = height + "px";
```

Comment:
we should decide if we use jquery or not across the project, mixing scripts with/out jquery can cause a mess since jquery keeps some values on its own "state".

Reply:
my opinion is to avoid using jQuery when possible, I am aware that we are not going to get rid of jQuery anytime soon, but the browser APIs are powerful enough now to barely need jQuery anymore.
Furthermore, in this project we mostly use YUI, don't have to say much about why we should avoid using it (it is pretty lame)